.gitignore

```diff
@@ -9,7 +9,7 @@
 *.test
 *.xml
 *.swp
-*.iml
+/*.iml
```

Comment:
nit: I would just remove this.

Reply:
I'm not sure if I want to check in IntelliJ project files.